lib/src/main/java/com/smileidentity/SmileID.kt

```diff
@@ -159,6 +187,228 @@ object SmileID {
+     * excluding any pending jobs.
+     *j
+     * @return A list of strings representing the IDs of submitted jobs.
+     */
+    @JvmStatic
+    fun getSubmittedJobs(): List<String> = listJobIds(
+        includeSubmitted = true,
+        includeUnsubmitted = false,
+    )
+
+    /**
+     * Retrieves a list of unsubmitted job IDs.
+     * This method filters the job IDs to include only those that are still pending,
+     * excluding any completed (submitted) jobs.
+     *
+     * @return A list of strings representing the IDs of unsubmitted jobs.
+     */
+    @JvmStatic
+    fun getUnsubmittedJobs(): List<String> = listJobIds(
+        includeSubmitted = false,
+        includeUnsubmitted = true,
+    )
+
+    /**
+     * Initiates the cleanup process for a single job by its ID.
+     * This is a convenience method that wraps the cleanup process, allowing for a single job ID
+     * to be specified for cleanup.
+     *
+     * @param jobId The ID of the job to clean up.Helpful methods for obtaining job
+     *  *              IDs include: [getSubmittedJobs] [getUnsubmittedJobs]
+     */
+    @JvmStatic
+    fun cleanup(jobId: String) = cleanupJobs(jobIds = listOf(jobId))
+
+    /**
+     * Initiates the cleanup process for multiple jobs by their IDs.
+     * If no IDs are provided, a default cleanup process is initiated that may target
+     * specific jobs based on the implementation in com.smileidentity.util.cleanup.
+     *
+     * @param jobIds An optional list of job IDs to clean up. If null, the method defaults to
+     * a predefined cleanup process.  Helpful methods for obtaining
+     * job IDs include:[getSubmittedJobs], [getUnsubmittedJobs]
+     */
+    @JvmStatic
+    fun cleanup(jobIds: List<String>? = null) = cleanupJobs(jobIds = jobIds)
+
+    /**
+     * Submits a previously captured job to SmileID for processing.
+     *
+     * @param jobId The unique identifier for the job to be submitted. This ID should be obtained
+     *              through the appropriate SmileID service mechanism and is used to track and
+     *              manage the job within SmileID's processing system. Helpful methods for
+     *              obtaining job  IDs include: [getSubmittedJobs] [getUnsubmittedJobs]
+     *
+     * Usage:
+     * To use this function, ensure you are calling it from a coroutine scope or
+     * another suspend function. For example, in a coroutine scope:
+     *
+     * ```kotlin
+     * coroutineScope {
+     *     SmileID.submitJob("your_job_id")
+     * }
+     * ```
+     * Note: Ensure that the jobId provided is valid and that your environment is properly set up
+     * to handle potential network responses, including success, failure, or error cases.
+     */
+    @JvmStatic
+    suspend fun submitJob(
+        jobId: String,
+        scope: CoroutineScope = CoroutineScope(Dispatchers.IO),
+        exceptionHandler: ((Throwable) -> Unit)? = null,
+    ): Job = scope.launch {
```

Comment:
Shouldn't we use the passed in exception handler here?
```suggestion
    ): Job = scope.launch(getExceptionHandler(exceptionHandler)) {
```